packages/aws-cdk-lib/aws-ec2/lib/windows-versions.ts

```diff
@@ -912,7 +912,7 @@ export enum WindowsVersion {
   WINDOWS_SERVER_2019_ENGLISH_CORE_BASE_2023_12_13 = 'Windows_Server-2019-English-Core-Base-2023.12.13',
   WINDOWS_SERVER_2019_ENGLISH_CORE_BASE_2024_01_16 = 'Windows_Server-2019-English-Core-Base-2024.01.16',
   WINDOWS_SERVER_2019_ENGLISH_CORE_BASE_2024_02_14 = 'Windows_Server-2019-English-Core-Base-2024.02.14',
-  /** @deprecated */
+  /** @deprecated - no longer supported */
```

Comment:
Not sure why or when `Windows_Server-2019-English-Core-ContainersLatest` and `Windows_Server-2019-English-Full-ContainersLatest` were removed, see [Windows AMI version history](https://docs.aws.amazon.com/AWSEC2/latest/WindowsGuide/ec2-windows-ami-version-history.html), but they are unavaialble:

```sh
$ aws ec2 describe-images --owners amazon --filters 'Name=name,Values=Windows_Server-2019-English-Core-ContainersLatest'
{
    "Images": []
}
$ aws ec2 describe-images --owners amazon --filters 'Name=name,Values=Windows_Server-2019-English-Full-ContainersLatest'
{
    "Images": []
}
```